acceptance/telemetry/upload-fails/script

```diff
@@ -0,0 +1,12 @@
+export DATABRICKS_CLI_TELEMETRY_PID_FILE=./telemetry.pid
+export DATABRICKS_CLI_TELEMETRY_UPLOAD_LOGS_FILE=./out.upload_process.txt
```

Comment:
could we use something a bit more memorable?

> standard_prefix_package_shortname
> DATABRICKS_CLI_TELEMETRY_PIDFILE
> DATABRICKS_CLI_TELEMETRY_LOGFILE

Reply:
Fixed.